samcli/commands/check/lib/command_context.py

```diff
@@ -85,3 +96,23 @@ def read_sam_file(self):
             sam_template = yaml_parse(sam_template.read())
 
         return sam_template
+
+
+def parse_template():
+    all_lambda_functions = []
+
+    # template path
+    path = os.path.realpath("template.yaml")
```

Comment:
Why is `template.yaml` hardcoded here instead of getting it from a CLI option?

Reply:
For now the template file needs to be located in the current directory. Haven't implemented the CLI option yet.

Reply:
Can you add a `TODO` note for this future implementation?